docs/modules/ROOT/pages/control-plane/split-architecture.adoc

```diff
@@ -1,17 +1,35 @@
 = Crossplane Split Architecture
 :page-aliases: reference/control-plane-pitfalls.adoc
 
-This page shows the pitfalls and gotchas that we need to keep in mind if we develop for a split architecture.
+This page contains descriptions about how we can operate services in a split architecture.
 
 .Overview AppCat Architecture
 image::appcat_control_cluster.drawio.svg[]
 
-== Provider Configs
+== RBAC and Service Accounts
+
+The connection between the control-plane and the service-clusters is 1:N.
+Each control-plane will need kubeconfigs to N service clusters.
+Each service-cluster will need kubeconfigs to one control-plane.
+
+To make provisioning the kubeconfigs easier, the component will pre-provision service accounts and the correct RBAC on each of the clusters.
+
+* On the control-plane: there's a service account named `appcat-service-cluster`, which service clusters can use to connect to the control-plane
+* On service-clusters: there's a service account named `appcat-control-plane` which the control-plane can use to connect to each service-cluster.
+
+Due to the nature of SYN, we can't create the kubeconfigs during compilation time. So each time a new service-cluster gets added to a control-plane, the kubeconfig has to be generated and put into Vault manually.
+
+The steps how to generate a kubeconfig from a service-account are documented in the https://docs.appuio.cloud/user/how-to/connect-gitlab.html#_configuring_appuio_cloud_as_a_kubernetes_cluster_in_gitlab_without_the_agent[APPUiO Docs].
+
+== Gotchas and Pitfalls
+This chapter shows the pitfalls and gotchas that we need to keep in mind if we develop for a split architecture.
```

Comment:
those lines provide no value to the reader, can we remove them?

Reply:
They make the rendered page a bit prettier. Without them, there will be two headings without any space between them.